### ISSUE TITLE
Implement unix read_pipe.is_reading() method

### ISSUE THREAD

**Issue by asvetlov** [python/cpython#82495]
BPO | [38314](https://bugs.python.org/issue38314)
--- | :---
Nosy | @asvetlov, @1st1, @progmatic-99, @callumquick, @benedwards14
PRs | <li>python/cpython#17042</li><li>python/cpython#17755</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-09-29.11:22:13.562>
labels = ['easy', 'type-feature', '3.9', 'expert-asyncio']
title = 'Implement unix read_pipe.is_reading() method'
updated_at = <Date 2019-12-30.13:53:39.286>
user = 'https://github.com/asvetlov'
```

bugs.python.org fields:
```python
activity = <Date 2019-12-30.13:53:39.286>
actor = 'callumquick'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['asyncio']
creation = <Date 2019-09-29.11:22:13.562>
creator = 'asvetlov'
dependencies = []
files = []
hgrepos = []
issue_num = 38314
keywords = ['patch', 'newcomer friendly']
message_count = 7.0
messages = ['353494', '353495', '353499', '353500', '355465', '355484', '355939']
nosy_count = 5.0
nosy_names = ['asvetlov', 'yselivanov', 'shubh07', 'callumquick', 'benedwards14']
pr_nums = ['17042', '17755']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue38314'
versions = ['Python 3.9']
```

</p></details>

**Comment by ggold7046** [comment]
Hello, is this issue still needs to be done ? I would like to do it as my first contribution. 

> The issue is easy, basically we need the following code plus a test
> 
> ```python
>     def is_reading(self):
>         return not self._paused and not self._closing
> ```

Could anybody guide me on how to do it ?

**Comment by zhengfeihe** [comment]
`is_reading` has already been implemented in this pr: #17425.
We can close this issue
